src/ops/ops.go

```diff
@@ -347,3 +348,12 @@ func (o *ops) GetMCSLogs() (string, error) {
 
 	return string(logs), nil
 }
+
+func (o *ops) UploadInstallationLogs() (string, error) {
+	command := "podman"
+	args := []string{"run", "--rm", "--privileged", "-v", "/run/systemd/journal/socket:/run/systemd/journal/socket",
+		"-v", "/var/log:/var/log", "docker.io/tsorya/logs_sender:latest", "logs_sender", "-tag", "agent", "-tag", "installer",
```

Comment:
nice image :)

Reply:
rega, so we have to build this one as well? where?

Reply:
nope it was bug